README.md

```diff
@@ -31,6 +31,62 @@ I've collected usernames of amazing stargazers :star: in Github. By using Github
 
 You need to submit Github Token to access Github API.
 
+## Getting Started
+##### Pre-Installation Requirements
+###### Node
+- Download and install latest stable version of [Node](https://nodejs.org/en/download/).
```

Comment:
I think we should specify what version are we using and add that to the package.json under `"engines"` too.